solutions/usaco-736.mdx

```diff
@@ -73,15 +73,81 @@ public class cownomics {
 
 </LanguageSection>
 
-## Solution 2
+## Solution 2 - Brute force
 
-Brute force through every position and determine whether there are any characters that appear both in a spotty cow and a plain cow at this position.
+We'll iterate through every character in every genome, and determine whether, for each position, there are any characters that appear in any genome of a spotted and plain cow. If there are, this position isn't a possible solution, so we can end this part of our search and move on to the next position to save on runtime. If there aren't, this is a possible solution, so add one to our solution storage variable. 
```

Comment:
I think "and determine whether, for each position, there are any characters that appear in any genome of a spotted and plain cow" is a bit confusing.